### PR TITLE
luci-app-https-dns-proxy: package name change, support for RFC8484 binary

### PR DESCRIPTION
**Pull request by stangri**
This luci app supports the new RFC8484-compatible package and more resolvers.

Depends on https://github.com/openwrt/packages/pull/10754.

Signed-off-by: Stan Grishin <stangri@melmac.net>

### CODE REVIEW

**Comment by tapper82** [comment]
luci-app-https-dns-proxy: Will not build in master. I think it's looking for the old package still.

**Comment by stangri** [comment]
Logs or it didn't happen.

**Comment by hnyman** [comment]
This has not been merged as there are conflicts.

File: applications/luci-app-https-dns-proxy/po/templates/https-dns-proxy.pot [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+#: luci-app-https-dns-proxy/luasrc/view/https-dns-proxy/buttons.htm:45
+msgid "Stop"
+msgstr ""
+
+#: luci-app-https-dns-proxy/luasrc/model/cbi/https-dns-proxy.lua:37
+msgid "Stopped"
+msgstr ""
+
+#: luci-app-https-dns-proxy/luasrc/model/cbi/https-dns-proxy.lua:22
+msgid "Uknown Provider"
```
Comment (ldir-EDB0) [inline applications/luci-app-https-dns-proxy/po/templates/https-dns-proxy.pot]: typo - should be 'Unknown Provider"

**Comment by stangri** [comment]
@hnyman how can I fix this conflicting files issue? I think they've been merged after I created this PR. The easiest way for me would be to create the new PR, please advise.

**Comment by hnyman** [comment]
Easiest might be to rebase and force-push this branch. 

git checkout feature_branch
git rebase master
<fix conflicts>
<mark resolved>
git push -f 

I think that you are actually destroying/removing all existing translations (*.po), instead of moving/renaming the files to reflect the new package name. The error is due that you are trying to delete the old contents of those files instead of the current contents. (the contents have been changed in a translation sync, or by additional weblate translations in the meanwhile, I guess)

But a better (=more polite) approach would be to rename/move the .po files to be under the renamed package structure. Then the already done translations would not be destroyed. I guess that most of the strings are still roughly the same.

**Comment by hnyman** [comment]
But naturally, also abandoning this PR and just creating a new PR is also simple, as you say.

**Comment by stangri** [comment]
> But a better (=more polite) approach would be to rename/move the .po files to be under the renamed package structure. Then the already done translations would not be destroyed. I guess that most of the strings are still roughly the same.

Good call on renaming the existing translations! I will start anew.

**Pull request closed without merging**